pkg/network/encoding/encoding.go

```diff
@@ -29,6 +30,11 @@ var (
 
 	cfgOnce  = sync.Once{}
 	agentCfg *model.AgentConfiguration
+	stats    = struct {
+		orphanAggregations pkgtelemetry.Counter
+	}{
+		pkgtelemetry.NewCounter("usm", "orphan_aggregations", []string{"protocol"}, ""),
+	}
```

Comment:
why do you need the struct definition here?

Reply:
Not required, but keeps it consistent with other uses of telemetry, and makes it easier to add more metrics in the future.